src/errors.rs

```diff
@@ -304,6 +304,8 @@ pub enum NotificationHandlingErrorCode {
     NodeUnavailable,
     /// The specificed in-progress swap couldn't be found.
     InProgressSwapNotFound,
+    /// A payment was expected, but it was not received.
```

Comment:
Is this comment necessary (for the docs)?
It doesn't add any information that cannot already be derived from the enum variants name.

On the other hand, you could actually provide some clarification here imo:
_Inconsistent payment data: A payment notification has been received, but the corresponding payment could not be retrieved from the node._

(Same is true for the other variants)

Reply:
Docs improved in [6569e5d](https://github.com/getlipa/lipa-lightning-lib/pull/1053/commits/6569e5df976161a85fd6b62e277bbc67201b5815)